pkgs/development/python-modules/vllm/default.nix

```diff
@@ -0,0 +1,141 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+, which
+, ninja
+, packaging
+, setuptools
+, torch
+, wheel
+, psutil
+, ray
+, pandas
+, pyarrow
+, sentencepiece
+, numpy
+, transformers
+, xformers
+, fastapi
+, uvicorn
+, pydantic
+, aioprometheus
+, writeShellScript
+
+, config
+
+, cudaSupport ? config.cudaSupport
+, cudaPackages ? {}
+
+, rocmSupport ? config.rocmSupport
+, rocmPackages ? {}
+, gpuTargets ? []
+}:
+
+let
+  version = "0.2.6";
+in
+buildPythonPackage {
+  pname = "vllm";
+  inherit version;
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "vllm-project";
+    repo = "vllm";
+    rev = "2832e7b9f92e2d1dd7dfe37951e5837c61d3db20";
```

Comment:
Why not using a tag directly instead of the commit hash?

Reply:
Because this is now not v0.2.6.
I'll update it later to the new release.